mod/deckorate_attribution/assets/style/reference.scss

```diff
@@ -1,3 +1,24 @@
 .TYPE-reference .RIGHT-Xsubject.card-editor {
   display: none
-}
\ No newline at end of file
+}
+
+.modal-header {
+  display: none;
+}
```

Comment:
Wikirate makes heavy uses of modals. This will make it so that the modal header doesn't appear anywhere on the site. In general, it's best not to add styles to a selector you didn't add yourself unless you have a very good sense of all the places the selector is used. (Although, of course, you can use that broader selector along side of your own selector).

In this case, for now you can just leave the header there, even if it's a little ugly and not what the mockup says. I will look into ways to address that later.